### PR TITLE
Test "cannot import namespace HttpData"

### PR DESCRIPTION
**Pull request by mvidner**
- This is the missing test for #45.
- https://trello.com/c/DMPCXbZs

If I apply this PR and revert the other fix, RPM build or `rake test:unit` will fail:
```
[    9s] Failures:
[    9s] 
[    9s]   1) Yast::HttpServerClient#main works?
[    9s]      Failure/Error: expect { subject.main }.to_not raise_error
[    9s] 
[    9s]        expected no Exception, got #<NameError: component cannot import namespace 'HTTPDData'> with backtrace:
[    9s]          # /home/abuild/rpmbuild/BUILD/yast2-http-server-4.2.2/src/lib/http-server/clients/main.rb:29:in `main'
[    9s]          # ./http_server_test.rb:15:in `block (4 levels) in <top (required)>'
[    9s]          # ./http_server_test.rb:15:in `block (3 levels) in <top (required)>'
[    9s]      # ./http_server_test.rb:15:in `block (3 levels) in <top (required)>'
[    9s] 
[    9s] Finished in 0.61652 seconds (files took 0.22047 seconds to load)
```

### CODE REVIEW

**Comment by mvidner** [comment]
~Spam~ Review Reward:
historic autumn landscape

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+srcdir = File.expand_path("../../src", __FILE__)
+y2dirs = ENV.fetch("Y2DIR", "").split(":")
+ENV["Y2DIR"] = y2dirs.unshift(srcdir).join(":")
+
+libdir = "#{srcdir}/lib"
+$LOAD_PATH.unshift(libdir)
```
Comment (jreidinger) [inline test/test_helper.rb]: why is this needed? require "yast" with modified Y2DIR should be enough.

File: test/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    ]
+  end
+end
+
+require "yast"
+require "yast/rspec"
+
+RSpec.configure do |c|
+  c.extend Yast::I18n # available in context/describe
+  c.include Yast::I18n
```
Comment (jreidinger) [inline test/test_helper.rb]: why you need translations in test?

Comment (mvidner) [reply test/test_helper.rb]: You're right, I don't, this is a copied test_helper from I forgot where, uniformity seemed better than optimality.

File: test/test_helper.rb (hunk 1 above)
Comment (mvidner) [reply test/test_helper.rb]: Oh right
